specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3819,6 +4011,10 @@
           "enum": [
             "analyzingInfixMatching"
           ],
+		  "x-ms-enum": {
```

Comment:
Why are you changing synonym maps?

Reply:
I run the validation that you suggested (autorest . --azure-validator=true) and there was a warning for this.

Reply:
Strange that we missed this earlier. Please check whether this affects the generated code in any way. If so, we'll have to rethink how we define this parameter. If not, then everything is fine.

Reply:
Ok. I will check the generated code.